.travis.yml

```diff
@@ -143,6 +143,7 @@ matrix:
     - env: ORCA_JOB=ISOLATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_DEV
     - env: ORCA_JOB=INTEGRATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_DEV
     - env: ORCA_JOB=ISOLATED_UPGRADE_TEST_TO_NEXT_MAJOR_DEV
+    - env: ORCA_JOB=ISOLATED_TEST_ON_CURRENT
```

Comment:
@panshulK I think you need to add another environment variable to PHP 8 job and use that here. Because we've one more job var: ISOLATED_TEST_ON_CURRENT and this will make that job as well in allowed failure.
cc: @chandan-singh7929